cdc/sink/ddlsink/mysql/mysql_ddl_sink.go

```diff
@@ -247,3 +255,46 @@ func (m *DDLSink) Close() {
 		}
 	}
 }
+
+// asyncExecAddIndexDDLIfTimeout executes ddl in async mode.
+func (m *DDLSink) asyncExecAddIndexDDLIfTimeout(ctx context.Context, ddl *model.DDLEvent) error {
+	done := make(chan error, 1)
+	// wait for 1 second at most
+	tick := time.NewTimer(1 * time.Second)
+	defer tick.Stop()
+	log.Info("async exec ddl start",
```

Comment:
the ddl must be add index, how about rewrite this to `async exec add index ddl start`